Remotion/Data/DomainObjects.UnitTests/Persistence/StorageProviderManagerTest.cs

```diff
@@ -22,6 +22,7 @@
 using Remotion.Data.DomainObjects.Persistence.Configuration;
 using Remotion.Data.DomainObjects.Persistence.Rdbms;
 using Remotion.Data.DomainObjects.Tracing;
+using Remotion.Development.UnitTesting;
 
 namespace Remotion.Data.DomainObjects.UnitTests.Persistence
 {
```

Comment:
since we have a ready-made NullPersistenceExtension Instance, please replace the usage with a stub and test that the instantiated provider got the stub instance. otherwise, one could hardcode the Null-Instance and we wouldn't find it in the  tests.